### ISSUE TITLE
I can't log in on Ubuntu 22.04 and error with: FUSE/libva

### ISSUE THREAD

**Issue by cuathro** [standardnotes/forum#1712]
### Context: 
I upgraded my operating system to Ubuntu 22.04.1. AppImage is running FUSE 2.x (libfuse2:amd64) and in the new version FUSE 3.x (libfuse3-3:amd64) is installed by default. I followed the steps from the [official Appimage blog](https://docs.appimage.org/user-guide/troubleshooting/fuse.html#setting-up-fuse-2-x-alongside-of-fuse-3-x-on-recent-ubuntu-22-04-debian-and-their-derivatives) and started FUSE 2.x.

### Problem:
StandardNotes application opens, I can edit notes and create notes. But when it comes time to log in, simply nothing happens. It asks me for username, password and even 2FA but no way.

I ran the application from the terminal and I got some errors that I don't know if they are related or not: 

```
$ ./standard-notes-3.23.83-linux-x86_64.AppImage 

Warning: Ignoring XDG_SESSION_TYPE=wayland on Gnome. Use QT_QPA_PLATFORM=wayland to run on Wayland anyway.
09:17:41.628 › Checking for update
extServer: Server started at http://127.0.0.1:45653
libva error: vaGetDriverNameByIndex() failed with unknown libva error, driver_name = (null)
[11115:0817/091743.803735:ERROR:sandbox_linux.cc(377)] InitializeSandbox() called with multiple threads in process gpu-process.
09:17:44.320 › Checking for update (already in progress)
09:17:44.776 › Update for version 3.23.83 is not available (latest version: 3.23.83, downgrade is disallowed).
[11115:0817/091747.842299:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 1 times!
[11115:0817/091753.013950:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 2 times!
BackupsManager: Data backup successfully saved: 2022-08-17T13-17-53.031Z.txt
[11115:0817/091755.062591:ERROR:gl_surface_presentation_helper.cc(260)] GetVSyncParametersIfAvailable() failed for 3 times!
```

And something else that happened, was by performing the following action: Quick Settings > Open Preferences > Clear workspace > Select the Delete 1 local backup file > Delete Workspace. The application closed and the following message appeared in Ubuntu notifications:

![BugUbuntu2204-C_2022-08-17 09-49-55](https://user-images.githubusercontent.com/68512577/185151153-6fdf4c06-df12-499e-ad9f-150ca93da948.png)

####  Versions installed and with this problem:

standard-notes-3.23.83-linux-x86_64.AppImage (lasted)
standard-notes-3.23.94-linux-x86_64.AppImage (beta)
standard-notes-3.23.95-linux-x86_64.AppImage (beta)

#### Operating system: 
Ubuntu 22.04.1 
_| Linux 5.19.1-051901-generic 
| x86_64 GNU/Linux_

**Comment by cuathro** [comment]
**Update:** After searching for more information, I read the post on the [Crafting Privacy Listed Blog](https://blog.standardnotes.com/35591/microupdate-resolution-regarding-server-status). And this line made me check the credentials: 

> For 2FA existence leaks, we [present a decoy 2FA prompt](https://standardnotes.com/help/81/why-am-i-asked-for-2FA-even-though-I-dont-have-it-enabled) randomly (but deterministically) for any email address, regardless of whether the account exists or not. In addition, some services only ask for 2FA only after your password is correctly verified. We take this a step further and do not allow password verification without the correct 2FA token first. This prevents outsiders from attempting to make password guesses without having the correct 2FA token.

**Yes, the password was incorrect.** Verifying the 2FA token before checking the password was something I was unaware of (but sounds very interesting). The problem was that a bad login or incorrect credentials message does not appear on the screen (after correctly putting in the 2FA token); perhaps an alert stating that you were unable to log in would be helpful.

I will report part of the error in another Issue if appropriate.
